### DISCUSSION TITLE
Importing Git Repository not working with GitHub

### DISCUSSION THREAD

**Discussion by carlamariap** [vercel/community#1073] [Help] [answered]
When continuing with GitHub I always and up in an error message from GitHub: "This page is taking too long to load. Sorry about that. Please try refreshing and contact us if the problem persists."

What's the issue?

**Comment by amyegan** [comment, marked as answer]
Hi @carlamariap! This is a known bug from GitHub side. You need to remove yourself as a collaborator from organizations with many repositories. The `cs50` and `me50` GitHub orgs seem to be the most commonly linked to this error because they are so large. Take a look at https://github.com/community/community/discussions/35026#discussioncomment-3878922 for more info.
